pb/xchain.proto

```diff
@@ -34,6 +34,7 @@ enum XChainErrorEnum {
     RWACL_INVALID_ERROR = 34;
     GAS_NOT_ENOUGH_ERROR = 35;
     TX_VERSION_INVALID_ERROR = 36;
+    TEXT_AUDIT_CHECK_NOT_APPROVED = 37;
```

Comment:
only add an error code for text audit? could this error code be more generic for auditing?

Reply:
If tx fails to pass text audit check interface, it's this error. Others are already included in the current types.